indexing-service/src/main/java/org/apache/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -379,6 +379,11 @@ public TaskStatus call()
                           command.add("true");
                         }
 
+                        if (task.doesntNeedLookups()) {
+                          command.add("--doesntNeedLookups");
```

Comment:
The parameter should be called `--loadLookups` instead, and should take values `all`, `none`, `onDemand`.

Reply:
renamed